libraries/Standard/src/test/cdf/utility/test_lower_inc_gamma.birch

```diff
@@ -1,8 +1,10 @@
 /*
-    Testing lower_inc_gamma
-*/
+ * Testing `lower_inc_gamma`
+ */
 
 program test_lower_inc_gamma(a:Real64, x:Real64) {
+    // note that this is not a robust test
+    // TODO(pranavs): Have a streamlined test for this function
```

Comment:
We can remove this test, unless the plan is to write a comprehensive one (is there any need?)

Reply:
Likewise the other tests. If there's a need to test with random arguments, or perhaps just test edge cases, we could include tests. Otherwise these functions are called as part of the cdf tests, which should reveal errors.

Reply:
I thought about this a bit, and when running some of the tests locally, I checked to make sure that all of the different regimes were tested. While I'm convinced that the cdf code stress tests these implementations, I did want to compare these against boost to make sure that things were okay.

Reply:
Ok, that's fine. So is the plan that we temporarily include tests that compare against Boost, and remove them later (given the intent of these changes is to ultimately remove dependency on Boost)?

Reply:
I think for now, we can test the cdf implementations and it should be fine. But maybe for added robustness, we can add tests against Boost as well in the future?